composer.json

```diff
@@ -15,13 +15,18 @@
 		"php": ">=5.6"
 	},
 	"require-dev": {
-		"dealerdirect/phpcodesniffer-composer-installer": "^0.6",
 		"roave/security-advisories": "dev-master",
-		"wptrt/wpthemereview": "^0.2"
+		"wptrt/wpthemereview": "*",
+		"dealerdirect/phpcodesniffer-composer-installer": "0.6",
+		"php-parallel-lint/php-parallel-lint": "^1.2"
 	},
 	"scripts": {
-		"check-cs": ["@php ./vendor/squizlabs/php_codesniffer/bin/phpcs -w"],
-		"fix-cs": ["@php ./vendor/squizlabs/php_codesniffer/bin/phpcbf"]
+		"check:cs": ["@php ./vendor/squizlabs/php_codesniffer/bin/phpcs -p"],
+		"check:compatibility": ["@php ./vendor/squizlabs/php_codesniffer/bin/phpcs -p --standard=phpcs.xml,PHPCompatibilityWP"],
+		"check:theme-review": ["@php ./vendor/squizlabs/php_codesniffer/bin/phpcs -p --standard=phpcs.xml,WPThemeReview"],
+		"lint:php": ["@php ./vendor/bin/parallel-lint --exclude .git --exclude vendor ."],
+		"checks": [ "@check:cs", "@check:compatibility", "@check:theme-review"],
```

Comment:
Aside from my above remark, I have bad experienced with combining checks like this with automated build checking as the error codes are not picked up correctly, so the build will always pass, even when checks are failing.

Reply:
Thanks for the hint!